liblangutil/Exceptions.h

```diff
@@ -197,8 +206,8 @@ class Error: virtual public util::Exception
 			return "info";
 		case Severity::Warning:
 			return "warning";
-		case Severity::Error:
-			solAssert(isError(_severity), "");
+		case Severity::Unknown:
+			return "unknown" case Severity::Error: solAssert(isError(_severity), "");
```

Comment:
I believe everything in here except this and lines 195-196 are formatting changes from IDE and is not required. Please remove them.

Reply:
Fixed 👍